astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 1ecac98d547729393ca54bcbe4f03cd4e1bfcd9a
+Subproject commit cca08a1755e703f9ff94e1f60a5b15d9521c128f
```

Comment:
Just revert the submodule to the previous version and then commit and do a fixup rebase:

```
$ cd astropy_helpers
$ git checkout 1ecac98d547729393ca54bcbe4f03cd4e1bfcd9a
$ cd ..
$ git add astropy_helpers
$ git commit -m "Fixup"
$ git rebase -i master
```
then a text editor will open with:

```
  1 pick dd95943 Begin building functions to calculate bounding box
  2 pick 8c766b9 Outline functionality of cal_flux_bounds helper functions.
  3 pick 3e192d1 Minor edits to helper function. Squash this later
  4 pick 6a13e45 Add bounding box parameters and default bounds for some models.
  5 pick 16e81f9 Implement bounding box attributes and properties, in progress.
  6 pick 4bef961 Define _bound property for models where appropriate.
  7 pick 8755039 Rework bounding box property.
  8 pick 2b1fc2e Add render_models2sky function.
  9 pick 123574b Implement render_model2sky function.
 10 pick 6bfe498 Fixup
 11 
 12 # Rebase 3f6de97..6bfe498 onto 3f6de97
 13 #
 14 # Commands:
 15 #  p, pick = use commit
 16 #  r, reword = use commit, but edit the commit message
 17 #  e, edit = use commit, but stop for amending
 18 #  s, squash = use commit, but meld into previous commit
 19 #  f, fixup = like "squash", but discard this commit's log message
 20 #  x, exec = run command (the rest of the line) using shell
 21 #
 22 # These lines can be re-ordered; they are executed from top to bottom.
 23 #
 24 # If you remove a line here THAT COMMIT WILL BE LOST.
 25 #
 26 # However, if you remove everything, the rebase will be aborted.
 27 #
 28 # Note that empty commits are commented out
```

(where the line numbers above are not part of the actual text)

Change line 10 `pick 6bfe498 Fixup` to read `fixup pick 6bfe498 Fixup`.  Then move it to just below line 1, so that the fixup is applied to the commit where the error was made:

```
  1 pick dd95943 Begin building functions to calculate bounding box
  2 fixup pick 6bfe498 Fixup
  3 pick 8c766b9 Outline functionality of cal_flux_bounds helper functions.
  4 pick 3e192d1 Minor edits to helper function. Squash this later
  5 pick 6a13e45 Add bounding box parameters and default bounds for some models.
  6 pick 16e81f9 Implement bounding box attributes and properties, in progress.
...
```

Then:

```
$ git push origin --force
```

On newer versions of git there should be less risk of this happening--it's better about automatically updating submodules when switching between branches.